scripts/scil_sh_to_aodf.py

```diff
@@ -55,39 +54,58 @@ def _build_arg_parser():
 
     add_sh_basis_args(p)
 
-    p.add_argument('--sphere', default='repulsion724',
+    p.add_argument('--sphere', default='repulsion200',
                    choices=sorted(SPHERE_FILES.keys()),
                    help='Sphere used for the SH to SF projection. '
                         '[%(default)s]')
 
-    p.add_argument('--method', default='bilateral',
-                   choices=['bilateral', 'cosine'],
+    p.add_argument('--method', default='unified',
+                   choices=['unified', 'cosine'],
                    help='Method for estimating asymmetric ODFs '
                         '[%(default)s].\nOne of:\n'
-                        '    \'bilateral\': Angle-aware bilateral '
-                        'filtering [1].\n'
-                        '    \'cosine\'  : Cosine-based filtering [2].')
+                        '    \'unified\': Unified filtering [1].\n'
+                        '    \'cosine\' : Cosine-based filtering [2].')
```

Comment:
Just for readability. I thinks it's nicer if you use " instead of '. Then, inside, you can simply use ' instead of `\'`.

Reply:
I had a hard time understanding this comment because I thought you wanted me to write _instead of_ somewhere. Anyway, got it !